### PR TITLE
{Release} Upgrade to Azure CLI 2.13.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,105 @@
...
+
+* Remove deprecated 'az iotcentral' command module (#15114)
+
+**Key Vault**
+
+* Support `--hsm-name` for `az keyvault key encrypt/decrypt` (#15218)
+
+**Lab**
+
+* Fix #14127: __init__() takes 1 positional argument but 2 were given (#14827)
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:69]:
```suggestion
* Fix #14127: `__init__()` takes 1 positional argument but 2 were given (#14827)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,105 @@
...
+
+* `az functionapp deployment source config-zip`: Fixed an issue where config-zip could throw an exception on success on linux consumption (#15174)
+* Bugfix: Better error messages for webapp commands (#15203)
+* `az appservice domain create, show-terms`: Add ability to create app service domain (#15173)
+* `az functionapp create`: Removed the preview flag from Java 11 when creating a new function app (#15351)
+* [BREAKING CHANGE] az webapp create, az webapp up - Update available webapp runtimes (#15356)
+
+**ARM**
+
+* `az ts`: Add new commands for template specs ; [ARM] az deployment : Add support for --template-spec -s (#14448)
```
Comment (Juliehzl) [inline src/azure-cli/HISTORY.rst:45]:
```suggestion
* `az ts`: Add new commands for template specs
* `az deployment` : Add support for --template-spec -s (#14448)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,105 @@
...
+
+**Compute**
+
+* Fix host group creation FD count limitation (#15316)
+* Add new command to support upgrading extensions for VMSS (#15238)
+* Fix a vmss update, disk attach/detach, extension set/delete issue (#14992)
+
+**HDInsight**
+
+* : az hdinsight create: add deprecate information for argument --public-networrk-access-type and --outbound-public-network-access-type (#15181)
```
Comment (Juliehzl) [inline src/azure-cli/HISTORY.rst:55]:
```suggestion
* `az hdinsight create`: add deprecate information for argument --public-networrk-access-type and --outbound-public-network-access-type (#15181)
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,105 @@
...
+* `az network application-gateway waf-policy managed-rule rule-set add`: Expose server error to user to give more intuitive hint message. (#15261)
+* `az network application-gateway waf-policy managed-rule rule-set update`: Support to change rule set type version. (#15261)
+
+**RDBMS**
+
+* Bugfix: az postgres flexible-server create Remove hardcoded API version from network client. (#15392)
+
+**Role**
+
+* Fix #15278: az role assignment list/delete: Forbid empty string arguments (#15282)
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:85]:
```suggestion
* Fix #15278: `az role assignment list/delete`: Forbid empty string arguments (#15282)
```

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,105 @@
...
+++++++
+
+**ACR**
+
+* NON-BREAKING: az acr helm - deprecation url upate (#15293)
+* Add logtemplate and systemtask changes for ACR Tasks (#15254)
+
+**AKS**
+
+* support virtual-node with aks create: `az aks create --enable-addons virtual-node` (#15129)
```
Comment (Juliehzl) [inline src/azure-cli/HISTORY.rst:16]:
```suggestion
* Support virtual-node with aks create: `az aks create --enable-addons virtual-node` (#15129)
```

**Review by haroldrandom — APPROVED** [review]

File: src/azure-cli/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,105 @@
...
+* Support add Spot node pool (#15311)
+
+**AMS**
+
+* Fix #14687: Mixed resource group and account name in command "az ams streaming-endpoint show" #14687 (#15251)
+
+**APIM**
+
+* api import: Update short summary (#15207)
+
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:29]: This PR isn't qualified to be included in the history. 

```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 7]
```diff
@@ -3,6 +3,105 @@
 Release History
 ===============
 
+2.13.0
+++++++
+
+**ACR**
+
+* NON-BREAKING: az acr helm - deprecation url upate (#15293)
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:11]: What is "NON-BREAKING"?

Comment (fengzhou-msft) [inline src/azure-cli/HISTORY.rst:11]:
```suggestion
* `az acr helm`: Update deprecation url (#15293)
```

File: src/azure-cli/HISTORY.rst [hunk 8]
```diff
@@ -3,6 +3,102 @@
...
+**ARM**
+
+* `az ts`: Add new commands for template specs
+* `az deployment` : Add support for --template-spec -s (#14448)
+
+**Compute**
+
+* Fix host group creation FD count limitation (#15316)
+* Add new command to support upgrading extensions for VMSS (#15238)
+* Fix a vmss update, disk attach/detach, extension set/delete issue (#14992)
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:48]: What issue?

Comment (houk-ms) [inline src/azure-cli/HISTORY.rst:48]:
```suggestion
* Fix the image reference is missing issue (#14992)
```

**Review by houk-ms — CHANGES_REQUESTED** [review]

**Review by jiasli — APPROVED** [review]

**Review by houk-ms — APPROVED** [review]

**Review by Juliehzl — APPROVED** [review]

File: src/azure-cli/HISTORY.rst [hunk 9]
```diff
@@ -3,6 +3,102 @@
...
+* `az acr helm`: Update deprecation url (#15293)
+* Add logtemplate and systemtask changes for ACR Tasks (#15254)
+
+**AKS**
+
+* Support virtual-node with aks create: `az aks create --enable-addons virtual-node` (#15129)
+* Add node image only option for CLI (#15250)
+* Expect kube-dashboard addon be disabled by default (#15267)
+* `az aks create/update`: Add LicenseType support for Windows (#15257)
+* Support add Spot node pool (#15311)
```
Comment (fengzhou-msft) [inline src/azure-cli/HISTORY.rst:20]:
```suggestion
* Support add Spot node pool (#15311)
* Honor addon names defined in Azure CLI (#15376)
```

**Pull request merged**